script/Ciel/Day 10/10_01_CIEL10_5_2.txt

```diff
@@ -504,17 +532,18 @@ In my state, I... understand this broken woman.
 [sha:ebb130c570c15609d259f8c9ffe228f063564dc0]{
 -- Page 409, Offset 28209.
 -- 「もう、泣いちゃってカッワイ～♥
-"Ohhh, you're crying, that's so cute～♥
+"Ooh, you're crying, that's so cuuute～♥
 }
 [sha:b4acba8df6429229781866937b21226c02530384]{
 -- Page 409, Offset 28210.
 -- だいじょうぶ、夜はこれからだもの。たっぷり楽しみましょう？
-It's okay, the night is only starting. We'll have a lot of fun!
+Don't worry, the night's still young.
+ We'll have plenty of fun together!
 }
 [sha:d82327e91bfb6499122c7defc7f824977b00b57e]{
 -- Page 409, Offset 28211.
 -- どんなに泣いても漏らしてもかまわないわ。
-It's fine, you can cry or pee yourself as much as you want, I don't mind.
+Go ahead and cry or piss yourself as much as you want, I won't mind.
```

Comment:
I think this is more like, "it doesn't matter how much you beg" kinda sentence...
Cry and wet yourself, I won't budge > Because > You're going in this box no matter what
Something like that

>Go ahead and cry and wet yourself to your heart's content. Doesn't matter to me.

>I mean, at the end of the day...

>You'll be going in here anyway, wailing all the while.

Something like that?